### ISSUE TITLE
refreshed-2.6.1/lib/get_navigation/src/routes/get_transition_mixin.dart(415,39): error G75B77105: Member not found

### ISSUE THREAD

**Issue by coolight7** [Aniketkhote/refreshed#22]
**ATTENTION: DO NOT USE THIS FIELD TO ASK SUPPORT QUESTIONS. USE THE PLATFORM CHANNELS FOR THIS. THIS SPACE IS DEDICATED ONLY FOR BUGS DESCRIPTION.**
**Fill in the template. Issues that do not respect the model will be closed.**

**Describe the bug**
Hi, 
refreshed-2.6.1 seem not support for flutter-3.22.0
There are some errors output when `flutter run`:
```
refreshed-2.6.1/lib/get_navigation/src/routes/get_transition_mixin.dart(415,39): error G75B77105: Member not found: 'CupertinoRouteTransitionMixin.isPopGestureInProgress'
```

**Reproduction code
NOTE: THIS IS MANDATORY, IF YOUR ISSUE DOES NOT CONTAIN IT, IT WILL BE CLOSED PRELIMINARY**
```
flutter run -d win
```
**To Reproduce**

1. flutter run -d win
2. See error

**Expected behavior**

**Screenshots**

**Flutter Version:**
flutter/3.22

**Refreshed Version:**
2.6.1

**Describe on which device you found the bug:**
windows

**Minimal reproduce code**
flutter run -d win

**Comment by aniketkhote9** [comment]
Fixed in refreshed 2.6.3

**Comment by Aniketkhote** [comment]
Fixed in refreshed 2.6.3

**Comment by coolight7** [comment]
Awesome!
